server/src/main/java/com/objectcomputing/checkins/services/team/Team.java

```diff
@@ -0,0 +1,61 @@
+package com.objectcomputing.checkins.services.team;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.model.DataType;
+
+@Entity
+@Table(name ="team")
+public class Team {
+
+    public Team(String name, String description) {
+                        this.name=name;
+                        this.description=description;
+                        }
+
+    public Team() {
+    }
+
+    @Id
+    @Column(name="uuid")
+    @AutoPopulated
+    @TypeDef(type=DataType.STRING)
+    private UUID uuid;
+
+    @NotNull
+    @Column(name = "name", nullable = false)
```

Comment:
could be wrong here, but I believe @NotNull and nullable=false cancel each other out

Reply:
@hwilliams314 `@NotNull` = `nullable = false`

Reply:
so should one be removed?

Reply:
one is for entity validation...and one is a column mapping definition, related but different.